SwahiliBox/swahilibox-events-api
# Ft implement user login

#### What does this PR do?
- Implements login route. It enables created users to login into the application

#### Description of Task to be completed?
- This pr solves issue #43 

#### How should this be manually tested?
- Pull the changes and run `yarn test:integration `to run the integration tests.
Start the server. Hiting `/login` endpoint should create an account successfully.

#### What are the relevant tasks or issues?
#43
